chatlearn/synchronizer/parameter_sync.py

```diff
@@ -960,6 +964,7 @@ def __init__(self, src_model, dst_model, group_name, frequency, error_signal):
         self._num_dst_hyper_expert_parallel = None
         self._actor2hep = {}
         self.sorted_send_actors_for_routed_experts = None
+        self.send_actors_to_regroup_experts = []
```

Comment:
is this variable only applicable for Qwen or for both Megatron and Qwen?

Reply:
both.  if not in megatron2vLLM case, this variable will not be called. A comment prefered?

Reply:
OK. It might be better to define his variable and related functions in the base class `ParameterSyncGroup` but not `ParameterSyncGroupwithHEP` from my perspective.

Reply:
his->this

Reply:
Besides, `clear_cache` should clear `send_actors_to_regroup_experts` by default.

Reply:
Expert regrouping only applied in HEP. It's better to define in HEP class. `send_actors_to_regroup_experts` called in `build_rank_mapping*` only, clear is not necessary, but this comment is accepted, modification will be pushed later.